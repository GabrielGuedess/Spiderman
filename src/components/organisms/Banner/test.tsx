import { renderWithTheme } from 'utils/tests/helpers';

import { Banner } from '.';

describe('<Banner />', () => {
  it('should render the heading', () => {
    const { container } = renderWithTheme(<Banner />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
