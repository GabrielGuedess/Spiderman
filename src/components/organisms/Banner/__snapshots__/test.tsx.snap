// Jest Snapshot v1, https://goo.gl/fbAQLP

exports['<Banner /> should render the heading 1'] = `
.c0 {
  position: absolute;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  left: 0;
  top: 0;
  width: 100%;
  height: 100%;
  z-index: -1;
  overflow: hidden;
}

.c0::after {
  content: '';
  position: absolute;
  inset: 0;
  background: linear-gradient( 90deg, #0b0c10 24.48%, rgba(11,12,16,0.8) 54.09%, #0c0c10 77.6% );
}

@media (min-width:1170px) {

}

@media (min-width:1440px) {

}

@media (min-width:1170px) {

}

@media (min-width:1170px) {

}

<div
  class="c0"
>
  <video
    autoplay=""
    loop=""
  >
    <source
      src="/video/spiderman.mp4"
    />
  </video>
</div>
`;
